docs/spec/v0.6/transports/can.md

```diff
@@ -15,50 +15,50 @@ This specification defines layer 3 (Network) and 4 (Transport) of the ThingSet P
 - Master-less operation
 - Automatic node ID assignment without central instance
 - Efficient useage of CAN ID and data bytes
-- Transport protocol to allow payload of more than 8 bytes
+- Transport protocol to allow payload of more than 8 bytes (classical CAN) or 64 bytes (CAN FD)
 - RTR frame is not allowed
-- Fixed bitrate of 500 kbit/s
+- Fixed bitrate of 500 kbit/s (arbitration) and 2 Mbit/s (data phase if using CAN FD)
```

Comment:
This is slightly ambiguous. The data rate is 500 kbit/s both for arbitration and for data in classical CAN mode.